skiko/buildSrc/src/main/kotlin/tasks/configuration/JvmTasksConfiguration.kt

```diff
@@ -295,13 +296,25 @@ fun SkikoProjectContext.createLinkJvmBindings(
                         "/ignore:4217"
                     )
                 )
+                // workaround for VS Build Tools 2022 (17.2+) change
+                // https://developercommunity.visualstudio.com/t/-imp-std-init-once-complete-unresolved-external-sy/1684365#T-N10041864
+                if (windowsSdkPaths.toolchainVersion >= VersionNumber.parse("14.32")) {
+                    addAll(
+                        arrayOf(
+                            "/ALTERNATENAME:__imp___std_init_once_begin_initialize=__imp_InitOnceBeginInitialize",
+                            "/ALTERNATENAME:__imp___std_init_once_complete=__imp_InitOnceComplete"
+                        )
+                    )
+                }
                 addAll(
                     arrayOf(
                         "/NOLOGO",
                         "/DLL",
                         "Advapi32.lib",
                         "gdi32.lib",
                         "Dwmapi.lib",
+                        "ole32.lib",
```

Comment:
Very ubiquitous if we're talking about Windows Desktop 😄 They've been there since... forever, I think you could find ole32.lib in Windows 9x and Propsys.lib in Windows XP. But since we only support Windows 10+, we could also replace most of them with `WindowsApp.lib` or `OneCoreUAP.lib` (ignore the UAP part, it just means that it includes APIs that are available on Desktop devices).